doc/network_topology_detection.rst

```diff
@@ -21,6 +21,10 @@ SNMP
 ----
 It is possible to ask an IO-Controller or IO-device (conformance class B or
 higher) about its neighbours, using the Simple Network Management Protocol (SNMP).
+
+SNMP agents are collecting information on individual devices, and the SNMP manager
```

Comment:
SNMP agents collect information, SNMP manager retrieves information

Reply:
Changed to "SNMP agents collect information on individual devices, and the SNMP manager retrieves information from them."